source/plugin/workspace/index.rst

```diff
@@ -4,8 +4,34 @@ Setting Up Your Workspace
 
 This section shows the basic setup for your IDE, to get yourself ready to develop Sponge plugins.
 
-Contents
-========
+Choosing a Plugin Identifier
```

Comment:
I'd personally like to move this out of the workspace index article somewhere else (before this one) so it can be referenced and linked separately, not really related to the workspace setup. Thoughts?